README.md

```diff
@@ -242,6 +242,29 @@ var imageURL = imagekit.url({
 https://ik.imagekit.io/your_imagekit_id/tr:h-300,w-400,l-image,i-ik_canvas,bg-FF0000,w-300,h-100,l-end/img/sample-video.mp4
 ```
 
+**5. Arithmetic expressions In transformations**
```

Comment:
Rewrite "Arithmetic expressions in transformations". Applicable for all PRs across projects.

Reply:
done